File: seminar_3/Python/HW_task3.py
# Напишите программу, которая принимает на вход число (N) и выдаёт таблицу кубов чисел от 1 до N.
# 3 -> 1, 8, 27 
# 5 -> 1, 8, 27, 64, 125

from math import pow

num = int(input("Введите число: "))

if num > 0:
    for i in range(1, num + 1):
        print(pow(i, 3), end="")
        if i < num:
            print(", ", end="")
else:
    print("Введено неверное число")

File: seminar_3/Java/HW_task3.java
// Напишите программу, которая принимает на вход число (N) и выдаёт таблицу кубов чисел от 1 до N.
// 3 -> 1, 8, 27
// 5 -> 1, 8, 27, 64, 125
package seminar_3.Java;

import java.util.ArrayList;
import java.util.Scanner;

public class HW_task3 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Введите число N: ");
        int n = input.nextInt();
        ArrayList<Integer> result = new ArrayList<>();
        for (int i = 1; i <= n; i++) {
            result.add((int) Math.pow(i, 3));
        }
        System.out.println(result);
    }
}
